cmake/Modules/LocalRav1e.cmake

```diff
@@ -51,6 +51,8 @@ else()
             GIT_REPOSITORY https://github.com/lu-zero/cargo-c.git
             GIT_TAG "${AVIF_CARGOC_GIT_TAG}"
             GIT_SHALLOW ON
+            # Patch from https://github.com/lu-zero/cargo-c/pull/453
+            PATCH_COMMAND git apply --ignore-whitespace "${AVIF_SOURCE_DIR}/ext/cargo.patch"
```

Comment:
You need `UPDATE_COMMAND ""` as well

Reply:
Done

Reply:
Yes, otherwise on update cmake will try to reapply the patch on the existing checkout of the depencency, which will fail the second time since the patch was already applied once. `UPDATE_COMMAND ""` tells it to not update but fetch a fresh copy every time.
That's my understanding at least.